tests/extmod/framebuf4.py

```diff
@@ -0,0 +1,68 @@
+try:
+    import framebuf
+except ImportError:
+    print("SKIP")
+    import sys
+    sys.exit()
+
+def printbuf():
+    print("--8<--")
+    for y in range(h):
+        print(buf[y * w // 2:(y + 1) * w // 2])
+    print("-->8--")
+
+w = 16
+h = 8
+buf = bytearray(w * h // 2)
+fbuf = framebuf.FrameBuffer(buf, w, h, framebuf.GS4_HMSB)
+
+# fill
+fbuf.fill(0xaf)
+printbuf()
+fbuf.fill(0xf0)
+printbuf()
+fbuf.fill(0x0a)
+printbuf()
+fbuf.fill(0xfa)
+printbuf()
+fbuf.fill(0x0f)
+printbuf()
+fbuf.fill(0x00)
+printbuf()
```

Comment:
I don't think all of these fill tests are needed... surely just fill(0x0f), fill(0xa0) would be enough?